user_docs/en/changes.t2t

```diff
@@ -33,6 +33,7 @@ This only worked for Bluetooth Serial ports before. (#14524)
 - NVDA no longer occasionally causes Mozilla Firefox to crash or stop responding. (#14647)
 - In Mozilla Firefox and Google Chrome, typed characters are no longer reported in some text boxes even when speak typed characters is disabled. (#14666)
 - You can now use browse mode in Chromium Embedded Controls where it was not possible previously. (#13493, #8553)
+- For symbols which do not have a symbol description in the current locale, the default English symbol level will be used. (#14558, #14417)

```

Comment:
@CyrilleB79 

I've rewritten your alternative proposal, can this be merged with this changelog entry?

>Punctuation will be reported according to the default English symbol level for symbols which do not have a symbol description in the current locale. (, #14558, #14417)

Reply:
> @CyrilleB79
> 
> I've rewritten your alternative proposal, can this be merged with this changelog entry?
> 
> > Punctuation will be reported according to the default English symbol level for symbols which do not have a symbol description in the current locale. (, #14558, #14417)

Yes, this description is correct; this PR can be merged. Thanks.